Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic.yml

```diff
@@ -0,0 +1,2791 @@
+    type: condition
+    task:
+      id: d03a6d37-76b9-40dd-83ac-a9d3e48de6da
+      version: -1
+      name: Is SIEM enabled?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      Azure Log Analytics:
+      - "14"
+      Qradar:
+      - "3"
+      Splunk:
+      - "2"
+    separatecontext: false
+    conditions:
+    - label: Qradar
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: inList
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: QRadar,QRadar_v2,QRadar v3
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+          ignorecase: true
+    - label: Splunk
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: SplunkPy
+                    ignorecase: true
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+          ignorecase: true
+    - label: Azure Log Analytics
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Azure Log Analytics
+                    ignorecase: true
+                accessor: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -610,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: d3f0aa8d-651f-493f-8084-ed98d5e99160
+    type: playbook
+    task:
+      id: d3f0aa8d-651f-493f-8084-ed98d5e99160
+      version: -1
+      name: QRadarFullSearch
```

Comment:
Please change back to the task name (it was changed to the playbook name you are using)

Reply:
Done